en/docs/install-and-setup/configuration-for-wso2-api-manager.md

```diff
@@ -1,69 +1,129 @@
+    ```tab="Example"
+    keytool -export -alias wso2carbon -file wso2carbon.crt -keystore <API-M_HOME>/repository/resources/security/wso2carbon.jks
     ```
 
-        Import the certificate using the command given below, to the trust store in the Microgateway runtime.
+    - Import the certificate using the command given below, to the trust store in the Microgateway runtime.
 
-    ``` java
+    ```tab="Example"
         keytool -import -trustcacerts -alias wso2carbon2 -file wso2carbon.crt -keystore <MGW_RUNTIME_HOME>/runtime/bre/security/ballerinaTruststore.p12
     ```
 
-        2.  Open the `           <MGW_RUNTIME_HOME>/conf/micro-gw.conf          ` file. Add the alias (e.g. wso2carbon2) to the truststore to corresponding configurations.
-        For example, to configure the JWT authentication for API Manager JWTs, add the following JWT issuer configuration to the `           micro-gw.conf          ` .
+2.  Open the `<MGW_RUNTIME_HOME>/conf/micro-gw.conf` file. Add the alias (e.g. wso2carbon2) to the truststore to corresponding configurations.
+    
+    For example, to configure the JWT authentication for API Manager JWTs, add the following JWT issuer configuration to the `micro-gw.conf` .
 
-    ``` java
+
+    ```tab="Example"
         [[jwtTokenConfig]]
-        issuer="https://localhost:9443/oauth2/token"
-        audience="http://org.wso2.apimgt/gateway"
-        certificateAlias="wso2carbon2"
-        validateSubscription = false
+            issuer="https://localhost:9443/oauth2/token"
+            audience="http://org.wso2.apimgt/gateway"
+            certificateAlias="wso2carbon2"
+            validateSubscription = false
     ```
 
-        You can now successfully invoke APIs through the Microgateway using the JWT tokens issued by WSO2 API Manager.
+You can now successfully invoke APIs through the Microgateway using the JWT tokens issued by WSO2 API Manager.
+
+## Configuring Runtime for Older API Manager Versions
+
+When you use Microgateway 3.2.0 runtime with older versions of WSO2 API Manager (2.6.0, 3.0.0 or 3.1.0), please do the below configurations in the `<MGW_RUNTIME_HOME>/conf/micro-gw.conf` file.
+
+#### Enable Legacy Key Manager
+
+```tab="Example"
+    [keyManager]
+    enableLegacyMode = true
+```
+
+This enables using the API Manager key validation service for opaque token validation. 
+
+#### Disable Event Hub Configuration
+
+```tab="Example"
+    [apim.eventHub]
+    enable = false
+```
+
+Apim Event hub is used for retrieving API, Application and subscription created in API Manager. This is only available in API Manager 3.2.0.
+
+#### Change the Throttling Server URL
+
+```tab="Example"
+    [throttlingConfig.dataRetriever]
+    serverUrl = "https://{API-M_HOST}:{API-M_PORT}/throttle/data/v1"
+```
+
+#### Change the Token Revocation Endpoint URL
+
+```tab="Example"
+    [tokenRevocationConfig.persistent]
+    endpointURL = "https://{API-M_HOST}:{API-M_PORT}/throttle/data/v1"
```

Comment:
Is this example or format of the config? {API-M_HOST}:{API-M_PORT} are not known properties

Reply:
Fixed in https://github.com/wso2/docs-mg/pull/40/commits/ce01f07681de7fbc80cd3d29cb95bf84c254aa45